pages/_app.js

```diff
@@ -0,0 +1,12 @@
+import "../styles/globals.css";
+import localFont from "@next/font/local";
+const neufileGrotesk = localFont({
+  src: "../public/assets/font/NeufileGrotesk-Regular.woff2",
+});
```

Comment:
Can you rename the fonts folder to `fonts`(in popular) as it can contain different fonts files